### ISSUE TITLE
About freeze specific layers with transfer learning

### ISSUE THREAD

**Issue by ntut108318099** [haotian-liu/yolact_edge#122]
Hi , @haotian-liu  Sorry to bother you.
I want to do transfer learning with mscoco and pascal voc . 
For now , I have done by using `--resume` command . The code can detect the dimension which is mismatch ,and then initialize it, and the model looks very well .
![image](https://user-images.githubusercontent.com/56862464/120589066-df7caf00-c46a-11eb-87bf-61e16564bf68.png)

But I want to freeze another layer (ex: several backbone layer ) to make the model more accuracy . What should I do ?

I also used `--drop_weights` command , but I think the command is used to make some specific layer's weight to delete .
The performance of  model didn't look very  well . It was also worse than the transfer model only use `--resume` command.   

BTW, I use the pytorch command about freezing parameters.

```sh
 yolact_net = Yolact()
net = yolact_net
for name, param in net.named_parameters():
        
        if  name.startswith("backbone") :
            param.requires_grad=False
            print("set_{}_frozen".format(name))  
net.train()
```
But the result didn't change a lot . Is the commend useful to the code ? 

Thanks for your reading.
Have a nice day.

**Comment by haotian-liu** [comment]
Hi, setting the parameters to `requires_grad=False` will freeze the backbone weights. Setting the `bn` in the backbone to eval mode is needed to prevent from bn statistics being updated.

**Comment by haotian-liu** [comment]
You can try using `fine_tune_layers` in the config (which will invoke the function that freezes the corresponding modules).

**Comment by haotian-liu** [comment]
I am closing this issue for now due to inactivity. If there are any further questions that I could help with, please feel free to reopen, thanks.
